README.adoc

```diff
@@ -349,12 +349,14 @@ following names:
 - `get:io.openliberty.guides.system.systemresource.getproperties`
 - `addtoinventory() span`
 
+// image::/guides/draft-guide-microprofile-opentracing/resources/inject-tracer.png[png][Inject tracer, width=100%]
+
 This simple example shows what you can do with the injected `Tracer` object. More configuration
 options are available to you, including setting a timestamp for when a span was created and destroyed.
 However, these options require an implementation of their own, which does not come as a part of the Zipkin
-user feature that is provided. In a real development environment, implement all the OpenTracing
-interfaces that you deem necessary, which might include the `SpanBuilder` interface. You can use this
-interface for span creation and customization, including setting timestamps.
+user feature that is provided. In a real world scenario, implement all the OpenTracing interfaces that
+you deem necessary, which may include the `SpanBuilder` interface. You can use this interface for span
```

Comment:
We try to avoid modal verbs, such as "may." Replace "may" with "might."

Reply:
Thats odd... it was "might" before. At some point I think a bunch of commits were reversed... :(

Reply:
Fixed